gengokit/httptransport/templates/server.go

```diff
@@ -262,4 +250,15 @@ func headersToContext(ctx context.Context, r *http.Request) context.Context {
 
 	return ctx
 }
+
+func HTTPDecodeLogger(next httptransport.DecodeRequestFunc, logger log.Logger) httptransport.DecodeRequestFunc {
```

Comment:
Basically, as a "just in case", but that may or may not be valid. Think it should be deleted?